modules/core/src/v2/coins/hbar.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * @prettier
+ */
+import * as Bluebird from 'bluebird';
+import { CoinFamily, BaseCoin as StaticsBaseCoin } from '@bitgo/statics';
+const co = Bluebird.coroutine;
+import * as bitgoAccountLib from '@bitgo/account-lib';
+
+import {
+  BaseCoin,
+  KeyPair,
+  ParsedTransaction,
+  ParseTransactionOptions,
+  SignedTransaction,
+  SignTransactionOptions,
+  VerifyAddressOptions,
+  VerifyTransactionOptions,
+  TransactionFee,
+  TransactionRecipient as Recipient,
+  TransactionPrebuild as BaseTransactionPrebuild,
+  TransactionExplanation,
+} from '../baseCoin';
+
+import { BitGo } from '../../bitgo';
+import { NodeCallback } from '../types';
+import { MethodNotImplementedError } from '../../errors';
+
+export interface HbarSignTransactionOptions extends SignTransactionOptions {
+  txPrebuild: TransactionPrebuild;
+  prv: string;
+}
+
+export interface TxInfo {
+  recipients: Recipient[];
+  from: string;
+  txid: string;
+}
+
+export interface TransactionPrebuild extends BaseTransactionPrebuild {
+  txHex: string;
+  txInfo: TxInfo;
+  feeInfo: TransactionFee;
+  source: string;
+}
+
+export interface ExplainTransactionOptions {
+  txHex?: string;
+  halfSigned?: {
+    txHex: string;
+  };
+  feeInfo: TransactionFee;
+}
+
+export class Hbar extends BaseCoin {
+  protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+
+  constructor(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>) {
+    super(bitgo);
+
+    if (!staticsCoin) {
+      throw new Error('missing required constructor parameter staticsCoin');
+    }
+
+    this._staticsCoin = staticsCoin;
+  }
+
+  getChain() {
+    return this._staticsCoin.name;
+  }
+
+  getFamily(): CoinFamily {
+    return this._staticsCoin.family;
+  }
+
+  getFullName() {
+    return this._staticsCoin.fullName;
+  }
+
+  getBaseFactor() {
+    return Math.pow(10, this._staticsCoin.decimalPlaces);
+  }
+
+  static createInstance(bitgo: BitGo, staticsCoin?: Readonly<StaticsBaseCoin>): BaseCoin {
+    return new Hbar(bitgo, staticsCoin);
+  }
+
+  /**
+   * Flag for sending value of 0
+   * @returns {boolean} True if okay to send 0 value, false otherwise
+   */
+  valuelessTransferAllowed(): boolean {
+    return false;
+  }
+
+  /**
+   * Checks if this is a valid base58 or hex address
+   * @param address
+   */
+  isValidAddress(address: string): boolean {
+    if (!address) {
+      return false;
+    }
```

Comment:
We should put this in `bitgoAccountLib.Hbar.Utils.isValidAddress()` if it isn't already. I know we check for `_.isEmpty()` but that doesn't include undefined I guess.

Reply:
sure, I'll add it to some cleanup we do there